src/engraving/libmscore/chordrest.cpp

```diff
@@ -917,6 +917,9 @@ EngravingItem* ChordRest::prevArticulationOrLyric(EngravingItem* e)
 
 EngravingItem* ChordRest::nextElement()
 {
+    if (nextChordRest(this)->tuplet()) {
+        return nextChordRest(this)->tuplet();
```

Comment:
Thank you! A couple of things:
1. Please don't call `nextChordRest` twice, it's quite a big function. Just store the result in a temp variable the first time.
2. `nextChordRest` may return null, so please null-check the output of the function before using it.